src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPaginatedItems(items: string[], perPage: number) {
+  const paginated = [];
+
+  for (let i = 1; i < Math.ceil(items.length / perPage) + 1; i++) {
```

Comment:
The loop condition calculation 'Math.ceil(items.length / perPage) + 1' is done on every iteration, which is inefficient. Consider calculating this once before the loop starts and storing it in a variable.

Reply:
fixed